display/d.histogram/bar.c

```diff
@@ -26,9 +26,10 @@
  */
 
 #include <string.h>
-
-#include <grass/raster.h>
```

Comment:
Please keep the empty line between the header groups, in case we decide to add [`IncludeBlocks(IBS_Preserve)`](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#includeblocks) to our ClangFormat definition.

Other than that minor detail, this looks good.